src/commands/controller.ts

```diff
@@ -1,17 +1,12 @@
 import type yargs from "yargs";
 import type { Arguments, CommandBuilder } from "yargs";
-import { ChainName, Registry } from "@tableland/sdk";
+import { Registry } from "@tableland/sdk";
 import { getWalletWithProvider, getLink } from "../utils.js";
-export type Options = {
-  // Local
+import { GlobalOptions } from "../cli.js";
+
+export type Options = GlobalOptions & {
```

Comment:
I think this is an example where the interface stuff is nicer.

```typescript
export interface Options extends GlobalOptions {
  name: string;
  controller: string;
}
```

Reply:
Agreed, updated Options on them all to be interface instead of type.